Hummingbird.docc/Tutorials/FluentUniverse/Fluent-1-Galaxy.tutorial

```diff
@@ -0,0 +1,121 @@
+@Tutorial(time: 15) {
+    @Intro(title: "Create a Hummingbird + Fluent application.") {
+        Create a simple web application using the Hummingbird template.
+        @Image(source: "hummingbird.png", alt: "Hummingbird logo")
+    }
+    
+    @Section(title: "Create your project") {
```

Comment:
Maybe split this into two sections "Create your project" and "Add Galaxy API"

Reply:
I would stick the Fluent steps in the "Create your project" or add a separate section "Add Fluent".